### DISCUSSION TITLE
Struggling to get my proxy working

### DISCUSSION THREAD

**Discussion by LaFours23** [jef/streetmerchant#2415] [Help]
I have PIA and I am trying to set up the socks5 proxy and I'm getting the error  net::ERR_NO_SUPPORTED_PROXIES

My proxy setup is 

Proxy_Protocol Socks5
Proxy_Address username:password@proxy-nl.privateinternetaccess.com (I also tried the ip address instead of the DNS)
Proxy_Port 1080

all the information I got from the PIA website, not sure if there is something else I need to do in the setup

**Comment by stjeffrey** [comment]
I tried with docker but couldn't get it to work.
If you are running the node version, just add a file called global.proxies in the root folder of the project and add your PIA link as
```
socks5://username:password@proxy-nl.privateinternetaccess.com:1080
```

**Comment by Mr-Tech-13** [comment]
Try using other proxy servers from freeproxylists.net

**Reply by V3rgE** [reply]
@Mr-Tech-13 I'm using freeproxylists.net for my PROXY_ADDRESS how do I format it? PROXY_ADDRESS=http://IP:PORT doesn't get me anywhere (Target Closed error). How am I supposed to do this? I have PROXY_PROTOCOL=http and PROXY_PORT=80
What am I missing?

**Reply by Mr-Tech-13** [reply]
Try this:
PROXY_ADDRESS=149.56.47.132
PROXY_PROTOCOL=http
PROXY_PORT=443

Paste that directly over the current one (BB will give errors)

**Reply by Mr-Tech-13** [reply]
In dotenv

**Comment by V3rgE** [comment]
Thanks so much! Now I'm getting this error: ERR_EMPTY_RESPONSE  for every
search
Example:
 error :: ✖ [walmart] sony sonyps5c ps5 console - net::ERR_EMPTY_RESPONSE
at https://www.walmart.com/ip/PlayStation5-Console/363472942

**Reply by Mr-Tech-13** [reply]
That means that the proxy server is down and you need to find a new one. Update it in dotenv by replacing the current with the new
